micrometer-core/src/main/java/io/micronaut/http/server/netty/InstrumentedEpollEventLoopGroupFactory.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2017-2019 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.netty;
```

Comment:
I don't think we should be reusing the same package across modules. I assume this was done because you needed to override package private classes? If that is the case then we would need to make changes upstream to allow what you want to do here.

Reply:
Should I do the PR on core-13.x or master?

Reply:
Makes sense in `master`

Reply:
I have a problem, the classes have been moved to another module and to a new package in master:
`io.micronaut.http.server.netty` -> `io.micronaut.http.netty.channel`

Reply:
I think the only way to keep compatibility with both 1.3.x and master is to move the instrumented event loop group factory classes to `micronaut-core`